src/webextension/manage/actions.js

```diff
@@ -87,11 +87,12 @@ function addItemStarting(actionId, item) {
   };
 }
 
-function addItemCompleted(actionId, item) {
+function addItemCompleted(actionId, item, interactive = false) {
   return {
     type: ADD_ITEM_COMPLETED,
     actionId,
     item,
+    interactive,
```

Comment:
It took me a bit to see what `interactive` meant, but I get it now.  I can't think of a better term, though.  It might help to have a comment somewhere with the intent, in case the "us" in six months struggle.